go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	golang.org/x/net v0.15.0
 )
 
+replace github.com/linode/linodego => github.com/linode/linodego v1.22.1-0.20230913213544-b58773254148
+
 require (
```

Comment:
I think this PR will need a `go get && go mod tidy` run to fix the linter error related to this 👍